ui/app/components/clients/chart-container.hbs

```diff
@@ -0,0 +1,44 @@
+{{! HBS display template for rendering title, description and stat boxes with a chart on the right }}
+
+<div class="chart-wrapper single-chart-grid" ...attributes>
+  <div class="chart-header has-bottom-margin-xl">
+    <h2 class="chart-title">{{@title}}</h2>
+    <p class="chart-description">
+      {{@description}}
+    </p>
+  </div>
+
+  {{#if @hasChartData}}
+    {{#if (has-block "subTitle")}}
+      <div class="chart-subTitle">
+        {{yield to="subTitle"}}
+      </div>
+    {{/if}}
+
+    {{#if (has-block "stats")}}
+      {{yield to="stats"}}
+    {{/if}}
+
+    {{#if (has-block "chart")}}
+      <div class="chart-container-wide">
+        {{yield to="chart"}}
+      </div>
+    {{/if}}
+
+    {{#if (has-block "legend")}}
+      {{yield to="legend"}}
+    {{/if}}
+
+  {{else}}
```

Comment:
Empty state and stat boxes are intentionally mutually exclusive because a separate component should be used to render just stats if the chart doesn't have data
<img width="1058" alt="Screenshot 2024-01-23 at 2 21 48 PM" src="https://github.com/hashicorp/vault/assets/68122737/70944e50-010b-48cc-9cbd-caf1bce3a4f8">